### ISSUE TITLE
iPhone 8 iOS 13.6

### ISSUE THREAD

**Issue by JCosswq** [danieltroger/telnetd_ramdisk#5]
I use an iPhone 8 bypassed with checkra1n on iOS 13.6, it ran out of storage a couple of days ago and began an infinite boot loop that I was unable to get out of, 64 of 64gb used. I haven’t touched it since then, I haven’t tried updating with iTunes which means I haven’t gotten error 14. Considering that this is new and all could I use this script in DFU mode to possibly delete larger files and boot back into normal mode, without having to update?

**Comment by JCosswq** [comment]
By the way the phone is not disabled

**Comment by JCosswq** [comment]
I remember that the phone took 15.14gb in the “other section” and 5gb in the system section

**Comment by danieltroger** [comment]
Hey man, it's AWESOME that you haven't tried updating!

In theory you should be able to delete large files with the script and it should just boot again, that's why I wrote it.

Practically speaking though you will probably not be able to delete any files, because deleting files *needs free space* on apfs filesystems.

Meaning it is impossible to delete something because there's no free space and it's impossible to get free space because it's impossible to delete something.

It could however be possible to extract your system partition to your computer, remove or format the system partition and through that creating free space allowing you to delete files from the user partition and then copy the system partition back.

It would be great if it worked and I could help you with it, but it's not guaranteed it will work and it needs much manual input. How technical are you?

**Comment by JCosswq** [comment]
I’m not that technical, but I have some valuable photos that I can’t risk losing, I was planning on buying a test phone and recreate the situation and try this but I still feel unsure, it would be awesome if someone else were able to do just that but I don’t want to lose hope, I know things like this are possible and this theory could be possibly successful as well

**Comment by JCosswq** [comment]
I also need to know if there has been someone who has had the same situation as me and has successfully been able to delete files and boot up again normally

**Comment by JCosswq** [comment]
I don’t know man I might need a video on how to do all of this because I don’t want to mess up

**Comment by JCosswq** [comment]
Could you please send me a video on how to set this up I’m getting really nervous with this and I have no idea how to do this please

**Comment by danieltroger** [comment]
> I’m not that technical, but I have some valuable photos that I can’t risk losing

hmm, is there another way? If so, try that first

> I also need to know if there has been someone who has had the same situation as me and has successfully been able to delete files and boot up again normally

no, not that I know of. Everyone tried to update first and fucked their sep-firmware.img4. One guy did not. He discovered the "no space left on device" when trying to remove files. He was too impatient to try to remove the system partition and wiped his data instead.

> Could you please send me a video on how to set this up I’m getting really nervous with this and I have no idea how to do this please

Would you be open to do some call where you can share your screen and I can tell you what to do? I feel like that would be more efficient

**Comment by JCosswq** [comment]
I’m open at any time, should I message you on what day to start? Also about ramdisk, doesn’t doing this process overwrite data on the iPhone? I also need to know if my data is still there on the phone and not corrupted data

**Comment by danieltroger** [comment]
> Also about ramdisk, doesn’t doing this process overwrite data on the iPhone

No. It's in the name, **RAM**disk, meaning it's only in the RAM, random access memory, and doesn't need any changes to the disk to run.

Deleting and putting back the system partition in hopes of freeing space though does change data on the iPhone.

> I also need to know if my data is still there on the phone and not corrupted data

I don't see any reason for it to be corrupted.

But I'm no data extraction expert. If you have any other means of getting your data, do so. My idea of removing the rootfs is really the last resort, it's something I'd just like to try before you give up and restore, as it _could_ work.

> I’m open at any time, should I message you on what day to start?

Yeah sounds good. I work during the week, so weekend is best for me if you have the patience. Otherwise in the evening after 18:00 would work. My timezone is Europe/Berlin.

**Comment by JCosswq** [comment]
Also if I were to do this would it show my storage used, for example (Total storage:64GB, used 58.0GB. Free 6GB)

**Comment by danieltroger** [comment]
Yes, you can see the disk usage from the RAMdisk.

**Comment by JCosswq** [comment]
I mean the storage from my iPhone

**Comment by danieltroger** [comment]
> I mean the storage from my iPhone
Ah sorry, me too.

I meant to say that when you have mounted the NAND of your phone you can see the free space of the filesystem, the files on it as well as the size of the files.

**Comment by JCosswq** [comment]
Does it matter also if the checkra1n I used was tethered, meaning that when I restart it, it goes away, and does it matter if the iPhone was iCloud bypassed, but I still set a passcode for it?

**Comment by danieltroger** [comment]
Shouldn't matter. But why did you icloud bypass it? 👀

**Comment by JCosswq** [comment]
What do I contact you on?

**Comment by danieltroger** [comment]
Is discord ok for you? I'm w4t3r#7442
